firmware/broadcom_trx.ksy

```diff
@@ -69,19 +77,19 @@ types:
         type: u1
 
   tail:
-    doc: "A safeguard against installation of firmware to an incompatible device"
+    doc: A safeguard against installation of firmware to an incompatible device
     seq:
       - id: version
         type: version
-        doc: "1.9.2.7 by default"
+        doc: 1.9.2.7 by default
       - id: product_id
         type: strz
         size: 12
       - id: comp_hw
         type: hw_comp_info
         repeat: expr
         repeat-expr: 4
-        doc: "0.02 - 2.99"
+        doc: 0.02 - 2.99
```

Comment:
Is this constraint worth checking with `valid/{min,max}`?

Reply:
I don't think so. The most likely it is just the values observed in the wild.